### ISSUE TITLE
Cannot Navigate Tree Items with Voice Over on Mac

### ISSUE THREAD

**Issue by chlafreniere** [microsoft/vscode#88181]
Issue Type: <b>Bug</b>

While having Voice Over enabled on Mac, cannot navigate tree items with the keyboard.

As an example, trying to use the arrow keys to navigate the File Explorer results in Voice Over reading one character at a time.

![image](https://user-images.githubusercontent.com/40371649/71860899-51e83480-30a9-11ea-8959-53e8b42dbf57.png)

Is there a way to workaround this issue?

VS Code version: Code 1.41.1 (26076a4de974ead31f97692a0d32f90d735645c0, 2019-12-18T14:57:51.166Z)
OS version: Darwin x64 18.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|3, 2, 2|
|Memory (System)|16.00GB (0.64GB free)|
|Process Argv|-psn_0_610453|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (12)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|2.0.11
gitlens|eam|10.2.0
tslint|eg2|1.0.44
nbpreviewer|jit|1.2.2
vscode-docker|ms-|0.9.0
mssql|ms-|1.8.0
sqlops-debug|ms-|1.3.0
python|ms-|2019.11.50794
csharp|ms-|1.21.9
vsliveshare|ms-|1.0.1404
debugger-for-chrome|msj|4.12.3
vscodeintellicode|Vis|1.2.2

</details>

**Comment by isidorn** [comment]
This works just fine for me.
Can you reproduce with VS Code insiders? https://code.visualstudio.com/insiders/
What are you doing differently than me?

![voiceOver](https://user-images.githubusercontent.com/1926584/71976727-503c7f00-3217-11ea-9e78-75c66e446395.gif)
